python/python/bystro/parent_of_origin/parent_of_origin.py

```diff
@@ -418,6 +422,29 @@ def fit(
                     )
                     ** 1.5
                 )
+                percentile_lower = 100 * norm.cdf(
```

Comment:
let's use percentile_lower and percentile_upper below, instead of repeating

```
100 * norm.cdf(
                    2 * z0_vector_component
                    + norm.ppf(alpha2 - a_vector_component)
                )
```

and

```
100 * norm.cdf(2 * z0_vector_component
                    + norm.ppf(alpha1 + a_vector_component)
                )
 ```
 
 also, are the boundaries 0 and 100 ok?

Reply:
0 and 100 are OK.